test/src/requestinterception-experimental.spec.ts

```diff
@@ -341,9 +341,13 @@ describe('cooperative request interception', function () {
         page.goto(server.EMPTY_PAGE).catch(() => {}),
       ]);
       expect(failedRequest).toBeTruthy();
-      expect(failedRequest.failure()!.errorText).toBe(
-        'net::ERR_INTERNET_DISCONNECTED'
-      );
+      if (isChrome) {
+        expect(failedRequest.failure()!.errorText).toBe(
+          'net::ERR_INTERNET_DISCONNECTED'
+        );
+      } else {
+        expect(failedRequest.failure()!.errorText).toBe('NS_ABORT_ERROR');
```

Comment:
This is wrong as BiDi does not support custom error like the test tries to test.

Reply:
WebDriver BiDi issue: https://github.com/w3c/webdriver-bidi/issues/508

Reply:
So should this test be skipped for both chrome and firefox, and ignored for now?

Reply:
The test only fail on assertion so I think it's fine to mark them as `FAIL` and update the comment to something like: 
`BiDi does not support custom errors - https://github.com/w3c/webdriver-bidi/issues/508`